source/tutorial/kerberos.txt

```diff
@@ -32,11 +32,11 @@ Kerberos authentication on either Windows or macOS.
 
          .. important::
 
-            The domain part of any username must be written 
-            in all capital letters. This part of the username 
-            corresponds to a Kerberos realm or Active 
-            Directory domain. It is case sensitive.
-            
+            The domain part of any username must be written
+            in all capital letters. This part of the username
+            corresponds to a Kerberos realm or Active
+            Directory domain. It *is* case sensitive.
+
          Active Directory Configuration
```

Comment:
I feel the Active Directory configuration should be above the Windows/macOS this is needed in both cases.
Can we also do a Linux section to complement the Windows and macOS sections?

Reply:
Per our conversation, we'll leave this for now and I added a note that the users should consult support for anything more than these two options (for now).